src/connectedk8s/azext_connectedk8s/custom.py

```diff
@@ -1874,11 +1873,11 @@ def client_side_proxy(cmd,
 
     # Fetching hybrid connection details from Userrp
     try:
-        list_prop = ListClusterUserCredentialsProperties(
+        list_prop = ListClusterUserCredentialProperties(
```

Comment:
I think we also decided in the last scrum to move CSP bits also to Azure CDN endpoint instead of the storage blob endpoint.
Also, the manifest rollout in FF will complete on monday night. We should not release before tuesday.

Reply:
made changes to use cdn endpoint for csp